.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+## Summary
+
+<!-- Please provide an overview of the work that was completed and the changes this PR introduces. Use the sections below as a guide to provide additional information. -->
+
+## Screenshots/Video
+
+<!-- If you're making UX changes, please provide screenshots. For UX bug fixes, make sure to provide before and after screenshots for comparison. For more complex changes, consider recording a video. -->
+
+## Testing
+
+<!-- Please provide instructions for how to test the PR that your reviewers can use! -->
+
+- Are there special case scenarios?
+- Are there configuration settings that need to be enabled locally?
+- Are there error cases to look out for?
+
+## Checklist
+
+<!-- Check each of the following boxes to assert that you've completed each task prior to publishing the PR. -->
+
+#### Basics
+
+- [ ] Does `npm run build` succeed?
+- [ ] Does `npm run test` succeed?
+
```

Comment:
```suggestion
- [ ] Are the commit messages meaningful?
```

Reply:
I'll add this to contributing.md, since it's something people should do before they make a PR rather than after.